share/picongpu/examples/FieldAbsorberTest/include/picongpu/param/fieldBackground.param

```diff
@@ -84,6 +86,39 @@ namespace picongpu
         /* Add this additional field? */
         static constexpr bool activated = true;
 
+        /* This setup is based on [Taflove, Hagness], section 7.11.1.
+         * The difference is we consider both 2D and 3D cases, and grid size may be increased due to our absorber being
+         * part of the simulation area, not located outside of it as in the book.
+         *
+         * Example of a rectangular conductor with a steady current.
+         *
+         * The conductor is oriented along the an axis.
```

Comment:
What do you mean by `an` axis?

Reply:
Good point, is a result of not fully done text editing